certified-connectors/RAPID Platform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,645 @@
+			"name": "RAPID Support",
+			"url": "http://www.rapidplatform.com.au",
+			"email": "support@rapidplatform.com.au"
+		}
+	},
+	"host": "api-test.rapidplatform.com",
+	"basePath": "/",
+	"schemes": [
+		"https"
+	],
+	"consumes": [],
+	"produces": [],
+	"paths": {
+		"/hooks": {
+			"x-ms-notification-content": {
+				"description": "Register a webhook on your RAPID site.",
+				"schema": {}
+			},
+			"post": {
+				"x-ms-trigger": "single",
+				"operationId": "ListHookTrigger",
+				"description": "Creates a webhook for when something happens in your RAPID site.",
+				"summary": "Trigger when something happens to an item in your RAPID site",
+				"consumes": [
+					"application/json"
+				],
+				"produces": [
+					"application/json"
+				],
+				"responses": {
+					"200": {
+						"description": ""
+					}
+				},
+				"parameters": [
+					{
+						"name": "Request body of webhook",
+						"in": "body",
+						"description": "Provides RAPID with the list being watched for changes and the trigger we are looking for.",
+						"required": true,
+						"schema": {
+							"type": "object",
+							"required": [
+								"config"
+							],
+							"properties": {
+								"config": {
+									"description": "The configuration of the Webhook to be registered.",
+									"required": [
+										"url",
+										"list",
+										"trigger"
+									],
+									"type": "object",
+									"properties": {
+										"url": {
+											"type": "string",
+											"description": "The location of this trigger that RAPID will update.",
+											"title": "Callback URL",
+											"x-ms-notification-url": true,
+											"x-ms-visibility": "internal"
+										},
+										"list": {
+											"type": "string",
+											"description": "Select a Table to be watched.",
+											"x-ms-summary": "Table",
+											"x-ms-dynamic-values": {
+												"operationId": "GetLists"
+											}
+										},
+										"trigger": {
+											"x-ms-summary": "Trigger Type",
+											"description": "Choose a webhook trigger event we are watching for.",
+											"type": "string",
+											"enum": [
+												"Item Created",
+												"Item Updated",
+												"Item Deleted"
+											]
+										}
+									}
+								}
+							}
+						}
+					}
+				]
+			}
+		},
+		"/openapi/lists": {
+			"get": {
+				"description": "Get Table titles from your RAPID site.",
+				"summary": "Get all Tables",
+				"operationId": "GetLists",
+				"x-ms-visibility": "internal",
+				"responses": {
+					"200": {
+						"description": "Returns a list of Table names currently registered to your RAPID site.",
+						"schema": {
+							"type": "array",
+							"items": {
+								"type": "string"
+							}
+						}
+					}
+				}
+			}
+		},
+		"/openapi/{listName}/schema": {
+			"get": {
+				"description": "Get the schema of one paticular item on your RAPID site.",
+				"summary": "Get the schema of selected Table",
```

Comment:
@MattFranklin-Rapid It looks like a duplicate one as per the summary value of next operation. Can you please update it accordingly.